sirepo/pkcli/admin.py

```diff
@@ -55,20 +80,50 @@ def create_examples():
         with sirepo.auth_db.session_and_lock(), auth.set_user_outside_of_http_request(
             uid
         ):
-            for sim_type in feature_config.cfg().sim_types:
-                names = [
-                    x.name
-                    for x in simulation_db.iterate_simulation_datafiles(
-                        sim_type,
-                        simulation_db.process_simulation_list,
-                        {
-                            "simulation.isExample": True,
-                        },
-                    )
-                ]
-                for example in simulation_db.examples(sim_type):
-                    if example.models.simulation.name not in names:
-                        _create_example(example)
+            callback()
+
+
+def reset_examples():
+    def _reset():
+        c = []
+        r = []
+        for sim_type in feature_config.cfg().sim_types:
+            sims = [
+                x
+                for x in simulation_db.iterate_simulation_datafiles(
+                    sim_type,
+                    simulation_db.process_simulation_list,
+                    {"simulation.isExample": True},
+                )
+            ]
+            _build_delete_and_revert_lists(c, r, sims, sim_type)
+        _revert_sims(r)
+        _delete_sims(c)
+
+    _access_sim_db_and_callback(_reset)
+
+
+def _build_delete_and_revert_lists(delete_list, revert_list, simulations, sim_type):
+    n = [n.models.simulation.name for n in simulation_db.examples(sim_type)]
+    for sim in simulations:
+        t = (
+            srtime.utc_now_as_milliseconds() - sim.simulation.lastModified
+        ) / _MILISECONDS_PER_MONTH
+        if sim.name not in n:
+            delete_list.append((sim, sim_type))
+        elif t > _MAXIMUM_SIM_AGE_IN_MONTHS:
```

Comment:
Make this:
  
```py
if _example_is_too_old(sim.simulation.lastModified):
```

This will simplify the test so you don't need a temporary.
    revert_list.